include/odp/api/spec/ipsec.h

```diff
@@ -772,6 +773,71 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * IPSEC SA information
+ */
+typedef struct odp_ipsec_sa_info_t {
+	/** Copy of IPSEC Security Association (SA) parameters */
+	odp_ipsec_sa_param_t param;
+
+	/** IPSEC SA direction dependent internal parameters */
```

Comment:
Looking at this patch now, I think the term "internal parameters" that appear here and elsewhere is not very clear. Why not simply "parameters"?

Reply:
Ack. Made it simply "parameters" in v9.